source/MaterialXCore/Node.cpp

```diff
@@ -140,15 +140,15 @@ Edge Node::getUpstreamEdge(size_t index) const
 
 OutputPtr Node::getNodeDefOutput(ElementPtr connectingElement)
 {
-    string outputName;
+    const string* outputName = nullptr;
```

Comment:
This change looks like an unintended revert of recent improvements to Node::getNodeDefOutput, so I'd recommend taking a closer look to see which of these changes are truly intended.